client/.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+    "env": {
```

Comment:
Should we also have an indent width of 2 in this file?

Reply:
It was automatically generated this way, but I can change it for the sake of consistency.

Reply:
In a way I feel that this is a decision to be made by the whole team ( Amount of indentation ), but we can look into it during a retro if it is deemed important later on. Lets just go with the 2 this time.